os/net/lwip/src/core/tcp_out.c

```diff
@@ -68,6 +68,7 @@
 #include <net/lwip/ipv4/inet_chksum.h>
 #include <net/lwip/stats.h>
 #include <net/lwip/snmp.h>
+#include <arpa/inet.h>
```

Comment:
Why include the header file here, if the rest of the file does not use it? Isn't it better to include files where they are used directly?

Reply:
After adding Werror flag we were getting 'ntohs' 'htonl' and 'htons' related error to overcome that error we have added this header file.